docs/sources/developers/kinds/core/star/schema-reference.md

```diff
@@ -0,0 +1,22 @@
+---
+keywords:
+  - grafana
+  - schema
+title: Star kind
+---
+> Both documentation generation and kinds schemas are in active development and subject to change without prior notice.
+
+# Star kind
+
```

Comment:
Should we have a statement about what Star kind does?

Reply:
Do you mean like a comment but for the entire Kind instead of specific fields description, right?

Is that something we have plans, for? @sdboyer @undef1nd

Reply:
Yes, it should be possible to have a kind description. But atm only field level comments are being exported from cue to json, and thus rendered in the docs.